src/NuGet.Clients/VisualStudio/Extensibility/IVsPackageInstaller.cs

```diff
@@ -31,10 +40,19 @@ public interface IVsPackageInstaller
         /// <summary>
         /// Installs a single package from the specified package source.
         /// </summary>
-        /// <param name="source">The package source to install the package from.</param>
+        /// <param name="source">
+        /// The package source to install the package from. This value can be
+        /// <c>"All"</c> or <c>null</c> to indicate that the user's configured
```

Comment:
I would leave 'all' out of this, it only worked before because VS 2013 understood it as a special source. Ideally callers should just pass null. All is a source that doesn't exist in 2015.